### ISSUE TITLE
Add `devenv processes tui` to connect to detached processes

### ISSUE THREAD

**Issue by tseaver** [cachix/devenv#971]

**Comment by domenkozar** [comment]
Maybe `devenv processes attach`? or `devenv processes foreground`?

**Comment by tseaver** [comment]
`devenv processes attach` is probably a clearer spelling, contrasting with the `--detach` option to `devenv proceses up` (but I admit to feeling warmer to `devenv processes foreground`).

**Comment by domenkozar** [comment]
After #2105 we should do a bit more work on this, currently there's no interactivity.

**Comment by rob-mur** [comment]
Ran into this when trying to upgrade today => is there currently no way to attach to processes launched with devenv process up -d (with the new manager)? You'd have to down and then up?

**Comment by pokonski** [comment]
Yeah that would be great to have, with process compose one could attach to the session, select a given process and use it interactively (for example inline debugging in Rails)

**Comment by domenkozar** [comment]
We'd need to make TUI protocol talk via unix socket. Quite a bit of mechanical work, I'm not going to tackle this soon but I'm happy to review a PR.

**Comment by domenkozar** [comment]
This is coming in 2.2 via https://github.com/cachix/devenv/pull/2890
